poc_v2.1_revised/vector_store.py

```
@@ -0,0 +1,186 @@
+        print(f"Removing existing vector store from {persist_dir}")
+        shutil.rmtree(persist_dir)
+
+    try:
+        embedding_model = OpenAIEmbeddings()
+        vector_db = Chroma.from_documents(
+            documents=chunks,
+            embedding=embedding_model,
+            persist_directory=persist_dir
+        )
+        return vector_db
+    except Exception as e:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)